must-gather/collection-scripts/gather_noobaa_resources

```diff
@@ -57,11 +57,18 @@ do
     done
 done
 
+# Collecting noobaa db pod logs with postgres label
+NOOBAA_POSTGRES_LABEL='noobaa-db in (postgres)'
+for pod in $(oc -n get pods -l "${NOOBAA_POSTGRES_LABEL}" | grep -v NAME | aws "{print $1}"); do
```

Comment:
```suggestion
for pod in $(oc -n get pods -l "${NOOBAA_POSTGRES_LABEL}" | grep -v NAME | awk '{print $1}'); do
```

Reply:
It should be double quotes, the operator CI failed saying that, this particular line should have double quotes and not a single quote.

Reply:
well we are using single quotes in must-gather scripts so can you try once with single quotes and then run `make shellcheck-test` to verify.

Reply:
Hi @rajatsing , I think crombus is pointing to your command itself (which should be `awk`). In the same script itself, there are other `awk` commands which have single quotes arguments...

Reply:
Oh yes!, you are right!. Thanks for pointing this out

Reply:
Updated, Thanks @aruniiird @crombus